lib/OnyxCache.ts

```diff
@@ -116,6 +117,21 @@ class OnyxCache {
         mergedKeys.forEach((key) => this.addToAccessedKeys(key));
     }
 
+    /**
+     * Allows to set all the keys at once.
+     * This is useful when we are getting
+     * all the keys from the storage provider
+     * and we want to keep the cache in sync.
+     *
+     * Previously, we had to call `addKey` in a loop
+     * to achieve the same result.
+     *
+     * @param keys - an array of keys
+     */
+    setAllKeys(keys: Key[]) {
+        this.storageKeys = new Set([...keys]);
```

Comment:
If `keys` argument is already an _iterable_, I think we can safely skip creating a new array and pass `keys` directly:

 ```ts
this.storageKeys = new Set(keys);
```

Let me know what you think 👍

Reply:
Yeah makes sense 👍